data/json/items/armor/hub_armor.json

```diff
@@ -0,0 +1,259 @@
+        "description": "Hooks for an armored skirt",
+        "flag_restriction": [ "ABLATIVE_SKIRT" ]
+      }
+    ],
+    "armor": [
+      { "encumbrance": [ 4, 6 ], "coverage": 100, "cover_vitals": 90, "covers": [ "torso" ], "layers": [ "Outer", "Belted" ] },
+      {
+        "encumbrance": [ 4, 4 ],
+        "coverage": 100,
+        "cover_vitals": 90,
+        "covers": [ "arm_l", "arm_r" ],
+        "specifically_covers": [ "arm_shoulder_l", "arm_shoulder_r", "arm_upper_l", "arm_upper_r" ]
+      },
+      {
+        "encumbrance": [ 4, 4 ],
+        "coverage": 100,
+        "cover_vitals": 90,
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r", "leg_upper_r", "leg_upper_l" ]
+      }
+    ]
+  },
+  {
+    "abstract": "robofac_mantle",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "abstract mantle",
+    "weight": "1 kg",
+    "volume": "1533 ml",
+    "price": 60000,
+    "price_postapoc": 100,
+    "material": [ "ceramic" ],
+    "symbol": ",",
+    "color": "dark_gray",
+    "material_thickness": 4,
+    "environmental_protection": 5,
+    "flags": [ "ABLATIVE_MANTLE", "CANT_WEAR" ],
+    "armor": [
+      { "encumbrance": 5, "coverage": 95, "covers": [ "torso" ], "specifically_covers": [ "torso_upper" ] },
+      {
+        "//": "skip encumbrance for additional parts of same body part",
+        "encumbrance": 0,
+        "coverage": 60,
+        "covers": [ "torso" ],
+        "specifically_covers": [ "torso_lower" ]
+      },
+      {
+        "encumbrance": 2,
+        "coverage": 95,
+        "covers": [ "arm_l", "arm_r" ],
+        "specifically_covers": [ "arm_shoulder_l", "arm_shoulder_r" ]
+      },
+      {
+        "//": "skip encumbrance for additional parts of same body part",
+        "encumbrance": 0,
+        "coverage": 85,
+        "covers": [ "arm_l", "arm_r" ],
+        "specifically_covers": [ "arm_upper_l", "arm_upper_r" ]
+      }
+    ]
+  },
+  {
+    "abstract": "robofac_skirt",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "abstract armored skirt",
+    "weight": "1 kg",
+    "volume": "900 ml",
+    "price": 60000,
+    "price_postapoc": 100,
+    "material": [ "ceramic" ],
+    "symbol": ",",
+    "color": "dark_gray",
+    "material_thickness": 4,
+    "environmental_protection": 5,
+    "flags": [ "ABLATIVE_MANTLE", "CANT_WEAR" ],
+    "armor": [
+      {
+        "encumbrance": 2,
+        "coverage": 100,
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ]
+      },
+      {
+        "//": "skip encumbrance for additional parts of same body part",
+        "encumbrance": 0,
+        "coverage": 85,
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_upper_l", "leg_upper_r" ]
+      }
+    ]
+  },
+  {
+    "abstract": "robofac_vambraces",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "abstract armored vambraces",
```

Comment:
```suggestion
    "name": { "str_sp": "abstract armored vambraces" },
```

Reply:
Sorry I didn't think abstract names needed plurals I'll fix this tomorrow. 

Also I appreciate you fixing all my spelling and formatting.